scalableminds/webknossos
# Circleci custom notification

### Steps to test:
- check messages such as https://scm.slack.com/archives/C3RUHRXBM/p1542638681073400
- the associated PR number will only be set in master (so I couldn’t test that yet. Will fix in follow-up if this won’t work
------
